### PR TITLE
Misc fixes for docker builds

### PR DESCRIPTION
**Pull request by ntsirakis**
Docker builds are working with the following changes. The only one I'm not 100% about is the build_env change; I wasn't able to find where it's supposed to be used, so I source it manually if the user uses the build command.

### CODE REVIEW

File: bordel [hunk 1]
```diff
@@ -84,6 +84,9 @@ fi
 if [ -d "${TOP}/${BUILD_DIR}" -a "${command}" != "config" ]; then
 	pushd "${TOP}/${BUILD_DIR}" >/dev/null
 fi
+if [[ "${command}" == "build" ]]; then
```
Comment (dpsmith) [inline bordel]: I would suggest moving this block to be at the beginning of `build_main` in cmds/build. It would be equivalent and drops the test.

**Comment by dpsmith** [comment]
Thank you so much for the contribution! Outside of the one comment, everything else looks good.

File: bordel (hunk 1 above)
Comment (ntsirakis) [reply bordel]: Done, not sure why I didn't do that in the first place lol. Also note that I add a certs generation step in the README; I didn't realize it was a mandatory step for some recipes.

Comment (dpsmith) [reply bordel]: Great, thanks!

**Review by dpsmith — APPROVED** [review]

**Pull request merged**